src/frontend/src/btc/stores/utxos-fee.store.ts

```diff
@@ -0,0 +1,34 @@
+import type { UtxosFee } from '$btc/types/btc-send';
+import type { Option } from '$lib/types/utils';
+import { writable, type Readable } from 'svelte/store';
+
+export type UtxosFeeStoreData = Option<{
+	utxosFee?: UtxosFee;
+}>;
+
+export interface UtxosFeeStore extends Readable<UtxosFeeStoreData> {
+	setUtxosFee: (data: UtxosFeeStoreData) => void;
+	reset: () => void;
+}
+
+export const initUtxosFeeStore = (): UtxosFeeStore => {
+	const { subscribe, set } = writable<UtxosFeeStoreData>(undefined);
+
+	return {
+		subscribe,
+
+		reset() {
+			set(undefined);
```

Comment:
I think we use `set(null)`. Can you double check?

Reply:
True, changed to `null`